### ISSUE TITLE
Question regarding training and pretrained model

### ISSUE THREAD

**Issue by vivekmadhavaram** [caiyuanhao1998/Open-DiffusionGS#20]
Hi @caiyuanhao1998,

I initialized the training using a pretrained checkpoint from Hugging Face and began training the model. At iteration 0, an output is saved for visualization; however, the rendered result shows all points lying along a straight line. I wanted to confirm whether this behavior is expected at initialization, or if the points should already be somewhat scattered in 3D space. Given that a pretrained checkpoint is loaded, I would expect at least a partially recognizable structure to appear. Could you clarify if this output is normal or indicative of an issue in the setup?

**Comment by caiyuanhao1998** [comment]
Hi, thanks for your interest,

`the rendered result shows all points lying along a straight line.` is not normal. It is an issue.

Since you use our model, I suggest you test before training to check that your environment installation has no problems.

If you find our repo useful, please help us star it, thanks :)

**Comment by vivekmadhavaram** [comment]
I tested with sample image and in testing process, the rendered views atleast preserved the structure but training fails. I have multiple views of the human face and camera parameters. I modified the data loader accordingly (no normal information and I am omitting the loss involving depth information). Currently, I use 6 views of person. I have following questions regarding this

Do I need to do all the data preprocessing you have done? (converting from blender world to OpenGL world)
Will there be any need of architectural change for human data? (I want to test as it is)
How much time usually the model takes to converge?
